.gitignore

```diff
@@ -1,4 +1,5 @@
-/target
+target/
+csspi/target/
```

Comment:
I believe that if we just specify `target` (`target/`?), it works.
[Hyper appears to be doing that](https://github.com/hyperium/hyper/blob/master/.gitignore#L1)

Reply:
Done